threescale_api/resources.py

```diff
@@ -43,7 +43,7 @@ def url(self) -> str:
 
 
 class Limits(DefaultClient):
-    def __init__(self, *args, entity_name='limit', entity_collection='limits', metric=None,
+    def __init__(self, *args, entity_name='limit', entity_collection='limits', metric,
```

Comment:
this looks like breaking change

Reply:
Well it can be breaking change but `PricingRules` has similar `__init__` with `metric = None`, see
```
77     def __init__(self, *args, entity_name='pricing_rule', entity_collection='pricing_rules',
  78                  metric: 'Metric' = None, **kwargs):
```
So I think we should be consistent here and choose the solution. What do you recommend?

Reply:
Are you sure? the change is exactly opposite to your example, it is removing default value of metric arg, and this is also that possibly breaking change